spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -39,6 +39,23 @@
     end
   end
 
+  context 'a belongs_to association with a presence validation and allow_nil' do
+    it 'allows a nil value' do
+      if ::ActiveModel::VERSION::STRING.to_f >= 4.0
+        child = define_parent_and_child_belongs_to_allow_nil.new
+        expect(child).to validate_presence_of(:parent).allow_nil
+      end
+    end
+
+    def define_parent_and_child_belongs_to_allow_nil(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `define_parent_and_child_belongs_to_allow_nil(*)` if you want the method to accept any arguments but don't care about them.